pkg/database/postgres_db_factory.go

```diff
@@ -0,0 +1,69 @@
+package database
+
+import (
+	"fmt"
+	glog "log"
+
+	log "github.com/sirupsen/logrus"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+	"gorm.io/plugin/dbresolver"
+)
+
+// postgresDbFactory will make Postgres DbInstance.
+type postgresDbFactory struct {
```

Comment:
sorry, but this is not a factory, this is PostgresAdapter or PostgressInstance or just `Postgres`.